client/src/selectors.js

```diff
@@ -0,0 +1,56 @@
+// @flow
```

Comment:
I think selector should have it's own folder. In the future we might want to split selectors across multiple files...

Reply:
yes -and once we have more then one we'll do it just like that :) it's a pattern I've been keeping also with other files - while there's just a single one it's in root, if there are multiple they're in appropriate folder